### PR TITLE
README: point to deployment template specs directly from github

### PR DESCRIPTION
**Pull request by marquiz**
User can directly use the (remote) template specs from github. No need
to download any files to do a deployment with the default setup as a
DaemonSet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,8 +190,8 @@ Feature discovery is preferably run as a Kubernetes DaemonSet. There is an
 example spec that can be used as a template, or, as is when just trying out the
 service:
 ```
-kubectl create -f rbac.yaml
-kubectl create -f node-feature-discovery-daemonset.json.template
+kubectl create -f https://raw.githubusercontent.com/kubernetes-incubator/node-feature-discovery/master/rbac.yaml
```
Comment (lmdaly) [inline README.md]: Will the files be downloaded by default when cloning the repo/getting a release?

Comment (marquiz) [reply README.md]: Yes, they are part of the repo and will be downloaded. But, the catch here is that you don't need to clone anything to deploy the latest release

**Comment by marquiz** [comment]
Rebased in order to adjust to the recent master/worker split

**Comment by Ethyling** [comment]
Should we point to the files on master or on the last release? As the docker images published do not match the current YAML files

**Comment by Ethyling** [comment]
Do we still need this PR?

**Comment by marquiz** [comment]
> Do we still need this PR?

No, I think this is outdated, now. Thanks for pointing this out

**Pull request closed without merging**